topics/authentication.md

```diff
@@ -6,140 +6,151 @@ Code examples: <a href="samples.md" anchor="authentication">Authentication</a>
 </p>
 </microformat>
 
-<include src="lib.xml" include-id="outdated_warning"/>
+Ktor provides the `Authentication` feature to handle authentication and access control. Typical usage scenarios include logging in users, granting access to specific resources, and secure transmitting information between parties. You can also use `Authentication` with [Sessions](sessions.md) to keep a user's information between requests.
 
+## Supported authentication types {id="supported"}
+Ktor supports different authentication schemes: from general HTTP authentication schemes such as Basic and Digest to OAuth 2.0 for accessing external resources.
 
-Ktor supports authentication out of the box. It supports mechanisms to read *credentials*, and to authenticate *principals*.
+### HTTP authentication {id="http-auth"}
+HTTP provides a [general framework](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) for access control and authentication. In Ktor, you can use the following HTTP authentication schemes:
+* [Basic](basic.md) - an authentication method to provide a user name and password encoded using `Base64`.
```

Comment:
an authentication method to provide a user name and password encoded using Base64 ->

uses Base64 encoding to provide a username and password. Generally is not recommended if not used in combination with HTTPS. 

(note that in this one you're using user name and below you're using username)